dynawo/CMakeLists.txt

```diff
@@ -346,7 +337,13 @@ endif()
 find_package(libXML 0.2.4 REQUIRED PATHS ${LIBXML_HOME})
 
 # libIIDM
-find_package(libIIDM 0.2.8 REQUIRED PATHS ${LIBIIDM_HOME}
+if ( CXX11_ENABLED )
+  set( libIIDM_RequiredVersion  "1.2.0")
+  find_package( LibXml2 REQUIRED)
+else ()
+  set( libIIDM_RequiredVersion  "0.2.8")
+endif ()
+find_package(libIIDM "${libIIDM_RequiredVersion}" REQUIRED PATHS ${LIBIIDM_HOME}
```

Comment:
On PowSybl website they recommend find_package(LibIIDM ...) (Uppercase L)

Reply:
not sure this is relevant though ;)

Reply:
I think this is the old code. In fact I prefer not to change it since that part is case-sensitive (it may be used within a unix file name). I noticed there is a confusion (between Libxml2, libxml2, LibXml2, and LIBXML2 too as another example). And rules change as you change libraries of course. In that case the old find_package( libIIDM... works with the new library. Changing to  LibIIDM may not work with old "finder"

Reply:
Please add this into a  if (CXX11_ENABLED) .. else () ... so that we can differentiate depending on which libIIDM library is used

Reply:
That is a good idea. Did you notice we use LibIIDM in libiidm.cmake new file ?

Reply:
Should be done now